py/torch_tensorrt/_Input.py

```diff
@@ -183,9 +183,9 @@ def __init__(self, *args: Any, **kwargs: Any) -> None:
                     kwargs["opt_shape"], dtype=kwargs["dtype"]
                 )
             elif self.shape_mode == Input._ShapeMode.DYNAMIC:
-                self.torch_tensor = self.example_tensor("opt_shape")
+                self.torch_tensor = self.example_tensor("opt_shape", zero_filled=True)
             else:
-                self.torch_tensor = self.example_tensor()
+                self.torch_tensor = self.example_tensor(zero_filled=True)
```

Comment:
If input is used as fake tensor and used as index, it caused cuda error. Input buffer is initialized as zero.